lib/appium_lib/driver.rb

```diff
@@ -329,6 +331,8 @@ def initialize(opts = {})
       @appium_device = @caps[:platformName]
       @appium_device = @appium_device.is_a?(Symbol) ? @appium_device : @appium_device.downcase.strip.intern if @appium_device
 
+      @automation_name = @caps[:automationName]
```

Comment:
Can you use down case such as [appium server](https://github.com/appium/appium/blob/9362fb224b18496425e0d7daa707d0035d8d9f7c/lib/appium.js#L54) ?
And could you also change `'XCUITest'` to `'xcuitest'` in this PR ?